frontend/src/components/ProfileContent.tsx

```diff
@@ -50,46 +61,107 @@ const useStyles = makeStyles({
         padding: 5,
         paddingRight: 10,
     },
+    bottomRight: {
+        position: 'absolute',
+        bottom: '0',
+        right: '0',
+        transform: 'translate(-10%, -10%)',
+    },
+    bottomRight2: {
+        position: 'absolute',
+        bottom: '0',
+        right: '0',
+        transform: 'translate(-100%, -10%)',
+    },
```

Comment:
I'm a little concerned about using `position: absolute` anywhere except in the most extreme cases.

Reply:
it's a powerful CSS style. as long as the correct parent has `position: relative` set, it simplifies the jsx

Reply:
I'm also a little leery of using 'absolute.' It's not the hill I'm going to do Tai Chi on, but it is something to pay attention to long term.

Reply:
Absolute messes with everything else on the page. How does it look on mobile, for example?

Reply:
I mean it looks about as good as everything else. I just don't think its worth fiddling with the exact position of things when we are just trying to get things to work.

![image](https://user-images.githubusercontent.com/6133706/107594447-ef346700-6bdf-11eb-9132-3aed972e31f3.png)